packages/node/src/constants.ts

```diff
@@ -0,0 +1,13 @@
+//API calls
+export const API_CALL_TOTAL_TIMEOUT = 29_000;
+export const API_CALL_TIMEOUT = 20_000;
+export const BLOCK_COUNT_HISTORY_LIMIT = 600;
+export const BLOCK_COUNT_IGNORE_LIMIT = 20;
+export const BLOCK_MIN_CONFIRMATIONS = 0;
+export const CONVENIENCE_BATCH_SIZE = 10;
+export const DEFAULT_RETRY_OPERATION_TIMEOUT = 5000;
+export const EVM_PROVIDER_TIMEOUT = 10_000;
+export const GAS_LIMIT = 500_000;
+export const OPERATION_RETRIES = 2;
+export const PROVIDER_INITIALIZATION_TIMEOUT = 20_000;
```

Comment:
I think for consistency and to indicate this is for a worker, let's rename to `WORKER_PROVIDER_INITIALIZATION_TIMEOUT` 

```suggestion
export const WORKER_PROVIDER_INITIALIZATION_TIMEOUT = 20_000;
```